pkg/scheduler/framework/score.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package framework
+
+import (
+	fleetv1 "go.goms.io/fleet/apis/v1"
+)
+
+// ClusterScore is the scores the scheduler assigns to a cluster.
+type ClusterScore struct {
+	// TopologySpreadScore determines how much a binding would satisfy the topology spread
+	// constraints specified by the user.
+	TopologySpreadScore int
+	// AffinityScore determines how much a binding would satisfy the affinity terms
+	// specified by the user.
+	AffinityScore int
+	// PriorityScore determines how much a binding would satisfy the priority terms
+	// specified by the user.
+	PriorityScore int
+}
+
+// Add adds a ClusterScore to another ClusterScore.
+func (s1 *ClusterScore) Add(s2 ClusterScore) {
```

Comment:
just curious when we'll need this func

Reply:
We need them as multiple plugins will return their own scores and it is up to the scheduler to add them together.

Reply:
oh, i see, thank for the info